### PR TITLE
Pin MSYS2 SDL2 version to 2.26.5

### PR DESCRIPTION
**Pull request by johnnovak**
Hacking the system 😎

### CODE REVIEW

File: .github/workflows/windows-msys2.yml [hunk 1]
```diff
@@ -91,13 +91,16 @@ jobs:
...
-            mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-SDL2
             mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-SDL2_net
             mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-zlib
             mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-speexdsp
             mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-python-pip
             mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-fluidsynth
 
+      - shell: msys2 {0}
+        run: |
+            pacman -U https://repo.msys2.org/mingw/${{matrix.conf.sys}}/mingw-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-SDL2-2.26.5-1-any.pkg.tar.zst
```
Comment (weirddan455) [inline .github/workflows/windows-msys2.yml:102]: You left out `w64` and it's failing with a 404 error.  Should be:

`pacman -U https://repo.msys2.org/mingw/${{matrix.conf.sys}}/mingw-w64-${{matrix.conf.toolchain}}${{matrix.conf.arch}}-SDL2-2.26.5-1-any.pkg.tar.zst`

Comment (johnnovak) [reply .github/workflows/windows-msys2.yml]: Thanks @weirddan455 👍🏻 Trying.

**Comment by Grounded0** [comment]
I feel like a twat editing the right version number but its my OCD again.

**Comment by johnnovak** [comment]
> I feel like a twat editing in the right version number but its my OCD again.

Haha, thanks. Being a coder, I should be better with number 😊

**Comment by johnnovak** [comment]
Okay, have tested this briefly, both the EXE and the ZIP package contain SDL 2.26.5 and they seem to work fine.

The high-DPI mouse position workaround @FeralChild64 put in there works nicely too; the pointer is in the middle of the window when uncapturing:
https://github.com/dosbox-staging/dosbox-staging/pull/2611/files

**Comment by johnnovak** [comment]
Tick please @kklobe @weirddan455 ?

**Review by weirddan455 — APPROVED** [review]
:+1:

**Pull request merged**